minimal_scheduling/minimal_sched_fifo_callback_group.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 Carlos San Vicente
+// Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
Is this file really based on a file by OSRF? If not, you should remove this line.
Applies to more files of this PR ...